src/xGadgetron/pGadgetron/tests/test_algebra.py

```diff
@@ -0,0 +1,68 @@
+#========================================================================
+# Copyright 2017 - 2021 Science Technology Facilities Council
```

Comment:
just 2021

Reply:
Also I removed the term SuperBuild which hasn't anything to do with this repo.